packages/key-management/src/util/createAsyncKeyAgent.ts

```diff
@@ -16,6 +16,7 @@ export const createAsyncKeyAgent = (keyAgent: KeyAgent): AsyncKeyAgent => {
     knownAddresses$,
     shutdown() {
       knownAddresses$.complete();
+      if (onShutdown) onShutdown();
```

Comment:
Alternative syntax (no need to change)
```suggestion
      onShutdown?.();
```

Reply:
Ah yeah, I just blindly did a rename refactor 🙈  I'll update it and squash at the same time